.github/ISSUE_TEMPLATE/Feature-request.yml

```diff
@@ -0,0 +1,54 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: ["Type: Feature request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        * We welcome any ideas or feature requests! It is helpful if you can explain exactly why the feature would be useful.
+        * There are usually some outstanding feature requests in the [existing issues list](https://github.com/espressif/arduino-esp32/issues?q=is%3Aopen+is%3Aissue+label%3A%22Type%3A+Feature+request%22), feel free to add comments to them.
+        * If you would like to contribute, please read the [contributions guide](https://docs.espressif.com/projects/arduino-esp32/en/latest/contributing.html).
+  - type: input
+    id: Area
+    attributes:
+      label: Related area
+      description: Please briefly explain the area of your Pull Request.
+      placeholder: eg. Board support, specific Peripheral, BT, Wifi...
+    validations:
+      required: true
+  - type: input
+    id: HW
+    attributes:
+      label: Hardware specification
+      description: Please provide if your proposal depends on specific Hardware.
+      placeholder: eg. Support for Adafruit 32u4 Board...
```

Comment:
Suggestion to add some DevKitC or any other Espressif Dev Board.

Reply:
Good point, edited.